modules/status-indicator/react/stories/stories_VisualTesting.tsx

```diff
@@ -36,7 +36,7 @@ export const StatusIndicatorStates = () => (
         ],
       })}
     >
-      {({type, emphasis, props}) => (
+      {({type, emphasis, ...props}) => (
         <StatusIndicator type={type} emphasis={emphasis} {...props} label="Status" />
```

Comment:
```suggestion
      {props => (
        <StatusIndicator {...props} label="Status" />
```

This works too right?

Reply:
Yep - I was getting type errors last time from this but I think it's because I forgot the `...`